lib/ui/tabs/home_widget.dart

```diff
@@ -120,11 +122,13 @@ class _HomeWidgetState extends State<HomeWidget> {
         debugPrint(
           "TabChange going from $_selectedTabIndex to ${event.selectedIndex} souce: ${event.source}",
         );
-        _pageController.animateToPage(
-          event.selectedIndex,
-          duration: const Duration(milliseconds: 100),
-          curve: Curves.easeIn,
-        );
+        if (_pageController.hasClients) {
```

Comment:
Why do we need this check?

Reply:
It was throwing errors if you turn on Uncaught Exceptions and run photos app.

Reply:
After checking, this exception seems to be coming from `FileSelectionOverlayBar` and not `HomeWidget` so removing this check.

Reply:
Update: @prateekmedia found that this is required for a different exception, so reverting removal of check.